installer/darwin/dmgconf.py

```diff
@@ -31,4 +31,5 @@ def get_size(start_path = '.'):
 #background = 'builtin-arrow'
 default_view = 'icon-view'
 
-
+format = 'UDZO'
+license = {'licenses': {'en_US': 'LICENSE.txt'}}
```

Comment:
Clearly it's not a serious issue, but the [dmgbuild docs](https://dmgbuild.readthedocs.io/en/latest/settings.html#license-settings) say that a 'default-language' key is also required in the license dictionary.